exercises/allergies/example.R

```diff
@@ -0,0 +1,34 @@
+allergy <- function(num) {
+  
+  allergy_list <- list(
+    cats = 128,
+    pollen = 64,
+    chocolate = 32,
+    tomatoes = 16,
+    strawberries = 8,
+    shellfish = 4,
+    peanuts = 2,
+    eggs = 1
+  )
+  
+  check_allergy <- function(allergy, val) {
+    if (val %% (allergy*2) >= allergy) {
```

Comment:
Well spotted! Turns out I didn't actually include that check in the linter. I should probably add it though. Will do a separate PR for that.